Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -24103,4 +24166,137 @@
+        <FlagArray index="TurnBeforeMove" value="1"/>
+        <FlagArray index="AISplash" value="1"/>
+        <FlagArray index="ArmySelect" value="1"/>
+        <PlaneArray index="Ground" value="1"/>
+        <Collide index="Colossus" value="1"/>
+        <Collide index="Structure" value="1"/>
+        <Attributes index="Armored" value="1"/>
+        <Attributes index="Biological" value="1"/>
+        <Attributes index="Massive" value="1"/>
+        <LifeStart value="1000"/>
+        <LifeMax value="1000"/>
+        <LifeArmor value="2"/>
+        <LifeRegenRate value="0.2734"/>
+        <LifeArmorName value="Unit/LifeArmorName/ZergGroundArmor"/>
+        <ShieldArmorName value="Unit/ShieldArmorName/ZergWildMutation"/>
+        <Speed value="2.9531"/>
+        <Acceleration value="1000"/>
+        <Sight value="9"/>
+        <Food value="-9"/>
+        <CostCategory value="Army"/>
+        <CostResource index="Minerals" value="600"/>
+        <CostResource index="Vespene" value="400"/>
+        <AttackTargetPriority value="20"/>
+        <AbilArray Link="stop"/>
+        <AbilArray Link="attack"/>
+        <AbilArray Link="move"/>
+        <AbilArray Link="AP_TyrannozorBurrowDown"/>
+        <AbilArray Link="AP_TyrannozorMergeable"/>
+        <AbilArray Link="AP_TyrannozorBarrageOfSpikes"/>
+        <AbilArray Link="ProgressRally"/>
+        <BehaviorArray Link="AP_MalignantCreepFix"/>
+        <BehaviorArray Link="AllUnitBehaviorController"/>
+        <BehaviorArray Link="AP_TyrannozorTyrantsProtectionCaster"/>
+        <BehaviorArray Link="AP_ImpalingStrike"/>
+        <BehaviorArray Link="Frenzy"/>
+        <BehaviorArray Link="AP_HealingAdaption"/>
+        <WeaponArray Link="AP_TyrannozorKaiserBlades"/>
+        <WeaponArray Link="AP_TyrannozorMissileLaunchers"/>
+        <CardLayouts>
+            <LayoutButtons Face="Move" Type="AbilCmd" AbilCmd="move,Move" Row="0" Column="0"/>
+            <LayoutButtons Face="Stop" Type="AbilCmd" AbilCmd="stop,Stop" Row="0" Column="1"/>
+            <LayoutButtons Face="MoveHoldPosition" Type="AbilCmd" AbilCmd="move,HoldPos" Row="0" Column="2"/>
+            <LayoutButtons Face="MovePatrol" Type="AbilCmd" AbilCmd="move,Patrol" Row="0" Column="3"/>
+            <LayoutButtons Face="Attack" Type="AbilCmd" AbilCmd="attack,Execute" Row="0" Column="4"/>
+            <LayoutButtons Face="Frenzied" Type="Passive" Behavior="Frenzy" Row="1" Column="4"/>
+            <LayoutButtons Face="BurrowDown" Type="AbilCmd" AbilCmd="AP_TyrannozorBurrowDown,Execute" Row="2" Column="4"/>
+            <LayoutButtons Face="AP_HealingAdaptation" Type="Passive" Requirements="AP_HaveHealingAdaptation" Row="1" Column="1"/>
+            <LayoutButtons Face="AP_TyrannozorTyrantsProtection" Type="Passive" Requirements="AP_HaveTyrannozorTyrantsProtection" Row="1" Column="2"/>
+            <LayoutButtons Face="AP_ImpalingStrike" Type="Passive" Requirements="AP_HaveImpalingStrike" Row="1" Column="3"/>

```

Comment:
Could you move the upgrade icons in order to start from the first column (index 0)? Frenzied should stay in place

Reply:
Fixed, wasn't sure about this, I figured it just looked better to have them all next to each other, but didn't want to move frenzied